### PR TITLE
[IMP] partner_delivery_schedule: Remove badge class to recovery font size

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT21941
@Tardo  @manuelcalerosolis  REview please...

### CODE REVIEW

**Review by Tardo — COMMENTED** [review]
You can use `$badge-font-size` and `$badge-font-weight` to control this...
BUT if you don't want change css, please, add a comment to know why this badges don't follow the bootstrap documentation.

**Comment by sergio-teruel** [comment]
Ok @Tardo Changes done!

**Comment by Tardo** [comment]
Please, check travis.

File: partner_delivery_schedule/views/report_shipping.xml [hunk 1]
```diff
@@ -3,7 +3,22 @@
...
+         -->
+        <style type="text/css">
+            .badge {
+                 font-size: 90%;
+            }
+        </style>
+    </template>
+
     <template id="report_picking" inherit_id="stock.report_picking">
+        <xpath expr="//div[@class='page']" position="inside">
```
Comment (Tardo) [inline partner_delivery_schedule/views/report_shipping.xml]:
```suggestion
        <xpath expr="//div[hasclass('page')]" position="inside">
```

File: partner_delivery_schedule/views/report_shipping.xml [hunk 2]
```diff
@@ -19,6 +34,9 @@
     </template>
 
     <template id="report_delivery_document" inherit_id="stock.report_delivery_document">
+        <xpath expr="//div[@class='page']" position="inside">
```
Comment (Tardo) [inline partner_delivery_schedule/views/report_shipping.xml]:
```suggestion
        <xpath expr="//div[hasclass('page')]" position="inside">
```

**Review by Tardo — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
@Tardo Done!!

**Review by Tardo — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**